aidies-project/routes/api/session.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const { check, validationResult } = require('express-validator');
+const { getUserToken, authenticated } = require('./auth');
+const bcrypt = require('bcryptjs');
+
+const { User } = require('../../db/models');
+
+const router = express.Router();
+
+const email =
+    check('email')
+        .isEmail()
+        .withMessage('Please provide a valid email address')
+        .normalizeEmail();
+        
+const password =
+    check('password')
+        .not().isEmpty()
+        .withMessage('Please provide a password');
+
+router.put('/', email, password, asyncHandler(async function (req, res, next) {
+    const { email, password } = req.body;
+    const user = await User.findOne({
+        where: { email },
+    });
+    const validatePassword = bcrypt.compareSync(password, user.hashedPassword.toString());
+    
+    if( !user || !validatePassword) {
+        const err = new Error("Login failed");
+        err.status = 401;
+        err.title = "Login failed";
+        err.errors = ["The provided email and password were invalid, please try again."];
+        return next(err);   
+    }
+    
+    const { jti, token } = getUserToken(user);/* 
+    user.tokenId = jti;
+    await user.save(); */
```

Comment:
Since we're not using the `jti`, we can update these lines:
```suggestion
    const token = getUserToken(user);
```